source/basic.tex

```diff
@@ -6169,9 +6169,15 @@
 \end{note}
 
 \pnum
-Two expression evaluations \defn{conflict} if one of them modifies a memory
+Two expression evaluations \defn{conflict} if one of them modifies\iref{defns.access} a memory
 location\iref{intro.memory} and the other one reads or modifies the same
 memory location.
+\begin{note}
+A modification conflicts with another expression evaluation
+at the same memory location
+even if the modification alters no bits in the object representation
+of the modified memory location.
```

Comment:
A memory location is not necessarily an "object", thus the term "object representation" might not be defined for it.

Suggestion for abbreviation: "Such conflicts occur even if a participating modification does not alter any bits."

Reply:
I agree that the old proposed wording was a bit too lengthy and still didn't achieve correctness.

The problem with *"Such conflicts"* is that we have defined what it means *to conflict* (verb), so ideally, we would not use the noun counterpart since it's ever so slightly informal.

I've settled on
> When a modification alters no bits, this does not prevent if from conflicting.